policyengine_us/parameters/calibration/social_security/outlay_history.yaml

```diff
@@ -0,0 +1,13 @@
+description: Historical total federal outlayers for social security (2017-2022)
```

Comment:
Could you clarify whether this is all of Social Security (OASDI, including retirement and disability)?